packages/react-ui-components/src/SelectBox_HeaderWithSearchInput/selectBox_HeaderWithSearchInput.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable camelcase, react/jsx-pascal-case */
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import mergeClassNames from 'classnames';
+
+const Fragment = props => props.children;
```

Comment:
Now that we have React 16 we could use the real fragment API?

Reply:
We need React 16.2.0 at least: https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html